fashify/src/pages/Auth/Login/Login.jsx

```diff
@@ -91,12 +72,18 @@ const Login = () => {
             >
               Login As a Guest
             </button>
-            <button
-              onClick={() => navigate('/signup')}
-              className='btn btn-link-primary outlined-secondary text-align-center brd-rd-semi-sq'
-            >
-              Create New Account
-            </button>
+            <div className='auth-footer'>
+              <p>
+                <span>Don't have an account?</span>
+                <span
```

Comment:
You can use the `<Link>` here rather than a span to avoid using navigate explicitly.